File: happy_birthday.py
from pylab import *

def sing_hb(name):
    song = ''
    hb = 'Happy Birthday'
    
    for i in range(4):
        if i != 2:
            song += '%s to you,\n' % hb
        else:
            song += '%s dear %s,\n' % (hb, name)
    
    return song


if __name__ == "__main__":
    
    print sing_hb('Jacob')

File: happy_birthday.m
function happy_birthday()

song = sing_hb('Jacob')

end


function song = sing_hb(name)
song = {};
hb = 'Happy birthday';
for i = 1:4
    if i ~= 3
        song{i} = sprintf('%s to you,',hb);
    else
        song{i} = sprintf('%s dear %s,', hb, name);
    end
end
song = char(song);
end